### ISSUE TITLE
"Standard" Module is Not Found. Breaks CM Operations.

### ISSUE THREAD

**Issue by populist** [pantheon-systems/drops-8#49]
There is logic in drupal_get_filename() which tries to see if the "standard" install profile is available. However, when the module list is created the "standard" value doesnt show up so the drupal_get_filename() check fails. This causes an error on the status report page and is part of a larger problem that breaks CM operations with this error:

    The configuration cannot be imported because it failed validation for the following reasons:
    Unable to install the standard module since it does not exist.

**Comment by tormi** [comment]
D.O issue: https://www.drupal.org/node/2486083

**Comment by populist** [comment]
This is still a problem with Drupal 8 Beta 14. Sigh.

**Comment by greg-1-anderson** [comment]
I am getting a very similar error during installation now.  Happens on the Pantheon platform with b10, b12 and b14, all of which previously worked.  The problem is similar, but in a different location:
```
unmet dependencies in Drupal\Core\Config\UnmetDependenciesException::create() (line 89 of core/lib/Drupal/Core/Config/UnmetDependenciesException.php).
Drupal\Core\Config\UnmetDependenciesException::create('standard', Array)
Drupal\Core\Config\ConfigInstaller->checkConfigurationToInstall('module', 'standard')
Drupal\Core\Extension\ModuleInstaller->install(Array, )
install_install_profile(Array)
install_run_task(Array, Array)
install_run_tasks(Array)
install_drupal(Object)
```
Seems this was fixed in core, but persists in Pantheon.  Is it getting worse, or is this related to #54? While it does not seem like these should necessarily be related, I am seeing the 'out of memory' error frequently now, whereas before I did not.

**Comment by dalibor-matura** [comment]
I'm getting the same error for Pantheon based site on page "admin/config/development/configuration" when running "Import all".

> The configuration cannot be imported because it failed validation for the following reasons:
> Unable to install the standard module since it does not exist.

**Comment by greg-1-anderson** [comment]
This is fixed by #61 + #17 + #59.

**Comment by greg-1-anderson** [comment]
The fix for this issue is included in the commits from the new reduced repository.  See #50.

**Comment by greg-1-anderson** [comment]
This actually was not completely fixed by the above issues.  I am working on a correct fix that should be ready shortly.  Also, this is just another symptom of #75.  I'm going to keep both open for now, though, since they appear to be different.

**Comment by greg-1-anderson** [comment]
Fixed by #77.

**Comment by greg-1-anderson** [comment]
Merged #77.
